apps/custom_media/locale/af/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,22 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
I'm not sure why you provide all locales. It generates a lot of work whenever we update a translation string. Why not only support languages that will have translated content?

Reply:
I generated them based on our `WAGTAIL_CONTENT_LANGUAGES`, but if we want to keep this to a minimum until someone translates the content to the other languages, I'm fine with that too

Reply:
Let's go with what you have now.

We can also drop languages in the future, if it is clear that they won't be used and maintaining empty languages becomes annoying.

Reply:
For this app in particular – I don’t think we need the `_("image")` verbose_name to be translate-able. Is that even displayed anywhere in the UI? This feels like a lot of files created for the single low-value string.

Reply:
> When several applications provide different versions of the same resource (template, static file, management command, translation), the application listed first in [INSTALLED_APPS](https://docs.djangoproject.com/en/4.1/ref/settings/#std-setting-INSTALLED_APPS) has precedence.

Via: https://docs.djangoproject.com/en/4.1/ref/settings/#installed-apps

So you can leave the string marked for translation, as Wagtail PO's will translate this string.
We only need to delete to remove the locale folder.